reference/promise-types/edit_line.markdown

```diff
@@ -436,11 +436,13 @@ mark out the region of a file to be edited.
      {
      select_start => "\[$(x)\]";
      select_end => "\[.*\]";
-     select_end_match_eof => "false";
+     select_end_match_eof => "true";
      }
 ```
 
 If the `select_end` attribute is omitted, the selected region will run to the end of the file no matter what the value of `select_end_match_eof` is set to.
+Please note, that there is also global `body agent control` [`select_end_match_eof][cf-agent#select_end_match_eof] configuration option which is used to set the default behavior of 
+`select_end_match_eof` for the entire policy. The local `edit_line` promise constraint is taking precedence over `body agent control` configuration option.
```

Comment:
The local `edit_line` promise constraint ~~is taking~~ **takes** precedence over **the** `body agent control` configuration option.